frontend/src/scenes/experiments/Metrics/PrimaryGoalTrends.tsx

```diff
@@ -31,70 +35,245 @@ export function PrimaryGoalTrends(): JSX.Element {
+                                    </div>
+                                    <div
+                                        className={`flex-1 cursor-pointer p-4 rounded border ${
+                                            currentMetric.exposure_query
+                                                ? 'border-primary bg-primary-highlight'
+                                                : 'border-border'
+                                        }`}
+                                        onClick={() => {
+                                            setExperiment({
+                                                ...experiment,
+                                                metrics: experiment.metrics.map((metric, idx) =>
+                                                    idx === metricIdx
+                                                        ? {
+                                                              ...metric,
+                                                              exposure_query: {
+                                                                  kind: NodeKind.TrendsQuery,
+                                                                  series: [
+                                                                      {
+                                                                          kind: NodeKind.EventsNode,
+                                                                          name: '$pageview',
+                                                                          event: '$pageview',
+                                                                      },
+                                                                  ],
+                                                                  interval: 'day',
+                                                                  dateRange: {
+                                                                      date_from: dayjs()
+                                                                          .subtract(EXPERIMENT_DEFAULT_DURATION, 'day')
+                                                                          .format('YYYY-MM-DDTHH:mm'),
+                                                                      date_to: dayjs()
+                                                                          .endOf('d')
+                                                                          .format('YYYY-MM-DDTHH:mm'),
+                                                                      explicitDate: true,
+                                                                  },
+                                                                  trendsFilter: {
+                                                                      display: ChartDisplayType.ActionsLineGraph,
+                                                                  },
+                                                                  filterTestAccounts: true,
```

Comment:
Does `dateRange` and `filterTestAccounts` need to be hard-coded? It seems like they should be derived from other configuration.

Reply:
`dateRange` should be okay as is - it's suitable for the preview (last 14 days), and it gets updated in the query runner when calculating results.

`filterTestAccounts` - probably a good idea..